server/api/paths.js

```diff
@@ -26,12 +26,12 @@ router.get('/all/user/:username/', async (req, res, next) => {
 // returns the most popular paths (regardless of category)
 // GET: api/paths/popular
 router.get('/popular', async (req,res,next) => {
-    const query = `MATCH (u: User)-[_p:PATHS]->(p: Path)<-[_r: REVIEWS]-(r:Review) 
-      RETURN p.name AS name, 
-             p.owner AS owner, 
-             count(r) AS reviewCount, 
-             count(distinct u) AS userCount, 
-             avg(r.score) AS rating 
+    const query = `MATCH (u: User)-[_p:PATHS]->(p: Path)<-[_r: REVIEWS]-(r:Review)
```

Comment:
Review count returned by query is not accurate.  I'm not sure if we're using it anywhere yet.
The query could be updated to something like:

MATCH (u: User)-[_p:PATHS]->(p: Path)
optional match (u)-[_r:REVIEWS]->(r:Review)-[:REVIEWS]->(p)
RETURN p.name AS name,
p.owner AS owner,
count(r) AS reviewCount,
count(distinct u) AS userCount,
avg(r.score) AS rating

for an accurate count.